refarch-backend/src/main/resources/application-local.yml

```
@@ -32,6 +34,5 @@ sso:
   client: local
 
 security:
-  # possible values: none, all, changing (With changing, only changing requests such as POST, PUT, DELETE are logged)
-  requestLogging: all
-  userInfoUri: ${sso.url}/auth/realms/${sso.realm}/protocol/openid-connect/userinfo
+  user-info-uri: ${sso.url}/auth/realms/${sso.realm}/protocol/openid-connect/userinfo
+  logging-mode: all
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)